main.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'securerandom'
+require 'json'
+# About memo app class
+class
+   Memo
```

Comment:
よく見たらここなんか変ですね。改行は必要ないですー

```suggestion
class  Memo
```

Reply:
確認ミスでした、修正しました